Docs: Terraria Setup Guide added information about the Upgraded Research Mod (#2338)
ArchipelagoMW/Archipelago: worlds/terraria/docs/setup_en.md

    - (Can be used to break progression)
- Reduced Grinding
- Upgraded Research

>     - (Automatically researchers certain items so don't be afraid when they don't appear in your inventory)

Reviewer: I find this wording to be awkward. I'd prefer it to be more straightforward and not mention being afraid.

>     - (Automatically researchers certain items so don't be afraid when they don't appear in your inventory)

Reviewer 2: ```suggestion
    - (Automatically researches certain items, removing them from your inventory initially)
```
Perhaps this might be a better wording

Author: I hope I did that right
